mintpy/objects/gps.py

```diff
@@ -217,11 +183,25 @@ def open(self, print_msg=True):
         self.read_displacement(print_msg=print_msg)
 
     def dload_site(self, print_msg=True):
+        # download time-series data from Nevada Geodetic Lab
+        url = 'http://geodesy.unr.edu/gps_timeseries'
+        if self.data_format == 'env3':
+            url = os.path.join(url, 'tenv3')
+        elif self.data_format == 'xyz2':
+            url = os.path.join(url, 'txyz')
+        url = os.path.join(url, 'IGS08/{}'.format(os.path.basename(self.file)))
+
         if print_msg:
-            print('downloading {} from {}'.format(self.site, self.file_url))
+            print('downloading {} from {}'.format(self.site, url))
+        urlretrieve(url, self.file)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Audit url open for permitted schemes. Allowing use of file:/ or custom schemes is often unexpected.](https://app.codacy.com/manual/insarlab/MintPy/pullRequest?prid=5671549)